aws/scenarios/microVMs/sample-vm-config.json

```diff
@@ -185,34 +185,36 @@
             "memory": [8192],
             "image": {
                 "image_path": "bullseye.qcow2.amd64-0.1-DEV",
-                "image_uri": "https://dd-agent-omnibus.s3.amazonaws.com/kernel-version-testing/rootfs/bullseye.qcow2.amd64-0.1-DEV.tar.gz"
-            }
-        },
-        {
-            "name": "custom-kernels-arm64-lte4.14",
-            "recipe": "custom-arm64",
-            "arch": "arm64",
-            "kernels": [
-                {
-                    "dir": "kernel-v4.14.arm64.pkg",
-                    "tag": "4.14",
-                    "extra_params": {
-                        "console": "ttyAMA0",
-                        "systemd.unified_cgroup_hierarchy": "0"
-                    }
-                }
-            ],
-            "vcpu": [4],
-            "memory": [8192],
-            "image": {
-                "image_path": "buster.qcow2.arm64-0.1-DEV",
-                "image_uri": "https://dd-agent-omnibus.s3.amazonaws.com/kernel-version-testing/rootfs/buster.qcow2.arm64-0.1-DEV.tar.gz"
+                "image_uri": "file:///home/kernel-version-testing/rootfs/bullseye.qcow2.amd64-0.1-DEV"
             }
         },
+        #{
```

Comment:
I would like to leave this in for now. Since the image for this particular kernel is not ready yet. I do not want the scenario to fail for people who are using it at the moment.